docs/rapidstart.md

```diff
@@ -1,8 +1,8 @@
-# Rapidstart \- Hello UC\!
+# Rapidstart
 
 This document provides an end-to-end installation and basic testing of all current components of Unity Catalog. It allows developers to try out the major aspects of Unity Catalog on their local machine. This rapid start will also be the basis for the full Unity Catalog Docker.
 
-# Quick Start
+## Quick Start
```

Comment:
quick start inside rapid start?

Reply:
We will be removing this rapidstart after adding the UI and auth pages.